ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/svc/TokenIssuerImpl.java

```diff
@@ -85,7 +85,7 @@ private boolean validateCode(Code code, String redirectUri, String clientId) {
     return code.clientId().equals(clientId);
   }
 
-  private String issueIdToken(String audience, String nonce, IdTokenJWS federatedIdToken) {
```

Comment:
I am fairly uncertain if that is a reasonable thing to do in Java, but I did not want to create the tests through the public methods because it felt incorrect.

Just let me know if my instincts are off here.